uber/templates/base.html

```diff
@@ -255,7 +255,10 @@
                 }
             };
             var message = '{{ message }}';
-            if (message.length) {
+            var error_message = '{{ error_message }}';
+            if (error_message.length) {
+                toastr.error(error_message);
```

Comment:
I added this `error_message` handler because I found the blue message box hard to notice on the `AT_THE_CON` registration page (it's particularly bad on the Labs server, because it has a blue background). I was worried that someone might try to register, forget to check the box, and then walk away from the kiosk, thus leaving all their personal information on the screen for the next person to see.

The checkbox would be achieving the opposite of the intended effect!